README.md

```diff
@@ -28,10 +28,17 @@ gulp.task('default', () =>
 
 ## API
 
-### gulpRemark()
+### gulpRemark([options])
 
 Gulp plug-in. It processes your files through [remark][remark]. If you want to define specified options, use [`.remarkrc`][remarkrc]. The ignoring of files available using [`.remarkignore`][remarkignore] file.
 
+#### options
+
+Type: `Object`  
+Default: `{}`
```

Comment:
You didn't understand.

I mean, remove default argument in `gulpRemark` function. Just make it `{}` too, because we don't need to set `silent`.

Otherwise, update description:

```diff
-Default: `{}`
+Default: `{ silent: false }`
```